django/db/backends/postgresql_psycopg2/base.py

```diff
@@ -231,6 +235,23 @@ def is_usable(self):
             return True
 
     @cached_property
+    def _nodb_connection(self):
+        nodb_connection = super(DatabaseWrapper, self)._nodb_connection
+        try:
+            nodb_connection.ensure_connection()
+        except (DatabaseError, WrappedDatabaseError):
+            warnings.warn(
+                "Unable to create a connection to the 'postgres' database, using "
```

Comment:
Unfortunately the drawbacks are mostly theoretical and not easy to explain briefly. Most of the time there probably is no drawback at all. I think an accurate explanation would go something like "This is probably just fine, but introduces a slight possibility that your default database could be modified during a test run, if something is mis-configured."

Maybe we can link to an explanation in the docs?